classes/Menu.php

```diff
@@ -365,7 +365,9 @@ public function generateReferences($page)
         };
 
         $iterator($items);
-
+        
+        Event::fire('pages.menu.referencesGenerated', [&$items]);
```

Comment:
You'll need to add inline event documentation in the form of https://github.com/octobercms/october/blob/master/modules/cms/classes/CmsObject.php#L150-L178 before I can accept this. Don't forget to provide a useful example of how to use this event.

Reply:
I apologize for this delay. Is this documentation sufficient?